inc/admin/metaboxes/namespace.php

```diff
@@ -66,8 +66,9 @@ function add_required_data( $pid, $post ) {
  *
  * @param $pid
  * @param $post
+ * @param $image
  */
-function upload_cover_image( $pid, $post ) {
+function upload_cover_image( $pid, $post, $image = null ) {
```

Comment:
Added the `$image` variable only for test purposes since will be `null` when using the application.